### PR TITLE
[18.09] Catch another Slurm memory cgroup creation error message

### PR DESCRIPTION
**Pull request by natefoo**

### CODE REVIEW

**Comment by martenson** [comment]
did you intend to replace `SLURM_UNABLE_TO_INSTANCIATE_JOB_MSG_RE` ?

**Comment by natefoo** [comment]
@martenson yeah, since the addition of `user` makes the old variable name somewhat inaccurate.

File: lib/galaxy/jobs/runners/slurm.py [hunk 1]
```diff
@@ -30,11 +30,11 @@
 SLURM_MEMORY_LIMIT_SCAN_SIZE = 16 * 1024 * 1024  # 16MB
 SLURM_UNABLE_TO_ADD_TASK_TO_MEMORY_CG_MSG_RE = re.compile(r"""slurmstepd: error: task/cgroup: unable to add task\[pid=\d+\] to memory cg '\(null\)'$""")
 SLURM_UNABLE_TO_CREATE_CGROUP_MSG_RE = re.compile(r"""slurmstepd: error: xcgroup_instantiate: unable to create cgroup '[^']+' : No space left on device$""")
-SLURM_UNABLE_TO_INSTANCIATE_JOB_MSG_RE = re.compile(r"""slurmstepd: error: jobacct_gather/cgroup: unable to instanciate job \d+ memory cgroup$""")
+SLURM_UNABLE_TO_INSTANCIATE_CGROUP_MSG_RE = re.compile(r"""slurmstepd: error: jobacct_gather/cgroup: unable to instanciate (job|user) \d+ memory cgroup$""")
```
Comment (nuwang) [inline lib/galaxy/jobs/runners/slurm.py:33]: Typo in 'instantiate'?

Comment (natefoo) [reply lib/galaxy/jobs/runners/slurm.py:33]: Nope, that's how Slurm spells it.

**Review by martenson — APPROVED** [review]

**Pull request merged**